utils/relay.py

```diff
@@ -149,7 +149,12 @@ def return_queryset_if_user_has_permissions(
     if is_customer(user):
         if customer_queryset is not None:
             return customer_queryset
-        return queryset.filter(customer__user=user)
+        if hasattr(queryset.model, "customer"):
+            return queryset.filter(customer__user=user)
+        elif hasattr(queryset.model, "owner"):
+            return queryset.filter(owner__user=user)
+        else:
+            raise Exception(f"Unsupported model {queryset.model}.")
```

Comment:
Maybe a `ValueError` or something a biiiit more specific? But up to you, lgtm otherwise 👍

Reply:
I guess it should be `TypeError` then. Changing to that.